gpii/node_modules/dbOperation/index.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+var fluid = require("infusion");
+
+fluid.module.register("dbOperation", __dirname, require);
```

Comment:
Rename this as gpii-db-operation - note that capital letters are not permitted in npm module names